Dockerfile.compute-node

```diff
@@ -0,0 +1,64 @@
+ARG COMPUTE_TOOLS_TAG=latest
+
+FROM debian:bullseye-slim AS build-deps
+RUN apt update &&  \
+    apt install -y git automake libtool build-essential bison flex libreadline-dev zlib1g-dev libxml2-dev \
+    libcurl4-openssl-dev libossp-uuid-dev
+
+# Build Postgres from the neon postgres repository.
+FROM build-deps AS pg-build
+RUN git clone -b main https://github.com/neondatabase/postgres.git postgres && \
+    cd postgres &&  \
+    ./configure CFLAGS='-O2 -g3' --enable-debug --with-uuid=ossp && \
+    make MAKELEVEL=0 -j $(getconf _NPROCESSORS_ONLN) -s install && \
+    make MAKELEVEL=0 -j $(getconf _NPROCESSORS_ONLN) -s -C contrib/ install && \
+    make MAKELEVEL=0 -j $(getconf _NPROCESSORS_ONLN) -s -C contrib/neon install && \
+    # Install headers
+    make MAKELEVEL=0 -j $(getconf _NPROCESSORS_ONLN) -s -C src/include install
+
+# Build PostGIS from the upstream PostGIS mirror. PostGIS compiles against neon postgres sources without changes.
+# Perhaps we could even use the upstream binaries, compiled against vanilla Postgres, but it would require some
+# investigation to check that it works, and also keeps working in the future. So for now, we compile our own binaries.
+FROM build-deps AS postgis-build
+COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
+RUN apt update && \
+    apt install -y gdal-bin libgdal-dev libprotobuf-c-dev protobuf-c-compiler xsltproc
+
+RUN git clone -b stable-3.2 https://github.com/postgis/postgis.git postgis && \
```

Comment:
Which tag would you recommend? Or would you recommend we build it around https://postgis.net/stuff/postgis-3.2.4dev.tar.gz instead of their git mirror?

Reply:
JFYI, the official link is https://download.osgeo.org/postgis/source/postgis-3.2.3.tar.gz

Reply:
Just saw they call "Latest stable branch" dev and "latest" is actually previous ❓ - will use the 3.2.3 tar as base